apps/src/sites/studio/pages/projects/public.js

```diff
@@ -14,7 +14,7 @@ $(document).ready(() => {
   registerReducers({projects});
   $.ajax({
     method: 'GET',
-    url: `/api/v1/projects/gallery/public/all/${MAX_PROJECTS_PER_CATEGORY}`,
+    url: `/api/v1/projects/gallery/public/all/${MAX_PROJECTS_PER_CATEGORY}?showFeatured=1`,
```

Comment:
I think this might override the "View all" categories as well (which I thought we still wanted as most recent).

Reply:
Also, I think we talked about this in person, but whatever solution you come up with here will also need to be done in https://github.com/code-dot-org/code-dot-org/blob/f4a791f903a4e5ea7f097a0338a1e1d5ec41b0df/apps/src/sites/studio/pages/projects/index.js#L52 , so that signed-in users will get the same experience. see `def public` and `def index` in projects_controller.rb for when each endpoint is used.

Reply:
Noted - once I get this in a good state, I'll find a way to share with index for signed in users.